version: '3.2'

services:
  local-db:
    build:
      context: .
      dockerfile: db.dockerfile
    image: hedza06/user-db:8.0.22
    ports:
      - "3307:3307"
    volumes:
      - ./data/db-data:/var/lib/mysql
      - ./data/db-config:/etc/mysql/conf.d
      - ./data/db-init:/docker-entrypoint-initdb.d/
    environment:
      MYSQL_ROOT_PASSWORD: root
      MYSQL_TCP_PORT: 3307
      TZ: 'Europe/Belgrade'
    restart: on-failure
    container_name: local_db
    networks:
      - cloudnet

networks:
  cloudnet: